etc/profile-a-l/bcompare.profile

```diff
@@ -0,0 +1,69 @@
+# Firejail profile for Beyond Compare by Scooter Software
+# Description: directory and file compare utility
+# Disables the network, which only impacts checking for updates.
+# This file is overwritten after every install/update
+
```

Comment:
```suggestion
```

No blank line here

Reply:
Yeah that's because there's no blank line in the template, but I think there *should* be a blank line in the template, to allow for separation between a user-friendly header, and the nitty-gritty of the profile.

Reply:
Yeah, that's become there's no blank line in the template, but I think there *should* be a blank line in the template, to separate the user-friendly header from the nitty-gritty of the profile.

Reply:
No profile has a blank line there, so if you think so you should open an issues IMHO. So we all can discuss this and change every profile in a separate PR.